src/System.Net.Http/tests/StressTests/HttpStress/corefx.Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Builds a dotnet sdk base image which contains corefx bits compiled from source
```

Comment:
Could you elaborate why this Dockerfile is needed and why you need to build corefx in a container yourself and then use that as the base of the other container? It seems like you could just have your build run in a hosted machine and then just copy the built binaries?

Reply:
Containerized pipelines do not support docker-in-docker so you can't create images when running your entire pipeline inside a container.

But beyond that reason, maintaing a Dockerfile enables reproducibility. I can use/debug this on my local dev machine without depending on AzDO.

Reply:
Makes sense. It just felt to me like it was too much overhead to have a Dockerfile here for just building corefx, but if it also simplifies local usage, I'm fine with having it.

Reply:
Right. I could have put everything into one single multi-staged dockerfile, however I felt that having a base image with custom bits is generally useful, hence the factoring.